tutorials/quickstarts/evals_quickstart.ipynb

```diff
@@ -0,0 +1,500 @@
+{
```

Comment:
<p>I think this is valuable but could be a separate tutorial in it of itself. This will fail without traces so I think we should split it out</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/4371/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>